src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,11 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const { name, email } = user;
```

Comment:
You should handle cases where user object is undefined or null to prevent errors.